package.json

```diff
@@ -36,7 +36,9 @@
     "test:p2p": "mocha --timeout 10000 -r ts-node/register test/p2p/*",
     "test:p2p:watch": "mocha --timeout 10000 -r ts-node/register test/p2p/*  --watch --watch-extensions ts",
     "test:crypto": "mocha --timeout 10000 -r ts-node/register test/crypto/*",
-    "test:sim": "(npm run compile:dev && cd test/simulation && export PATH=\"$PWD/go/bin:$PATH\" && GOPATH=$PWD/go GO111MODULE=on go test -timeout 30m -run=TestIntegration -v)",
+    "test:sim:build": "(npm run compile:dev && mkdir -p test/simulation/temp/xud && cp -a dist test/simulation/temp/xud && cp -a bin test/simulation/temp/xud && cp package.json test/simulation/temp/xud && cd test/simulation && ./docker-build.sh)",
+    "test:sim:clean": "(cd test/simulation && ./docker-clean.sh)",
+    "test:sim:run": "(cd test/simulation && ./docker-run.sh)",
```

Comment:
Can we just call this `test:sim`? I think that's straightforward enough and saves some typing.